rpm/sybase-systemd.spec

```diff
@@ -0,0 +1,73 @@
+Name:		sybase-systemd
+Version:	%{_provided_version}
+Release:	%{_provided_release}%{?dist}
+Summary:	Sybase systemd units
+License:	GPLv2
+URL:		https://github.com/hubertqc/selinux_sybase
+#Source:	%{name}-%{version}.tar.gz
+BuildArch:	noarch
+
+Requires:	systemd
+Requires:	sybase-selinux = %{version}-%{release}
+
+%description
+Definitions of systemd units for Sybase ASE.
+
+###################################
+
+%clean
+%{__rm} -rf %{buildroot}
+
+#%prep
+#%setup -q
+
+###################################
+
+%install
+
+mkdir -p -m 0755 %{buildroot}/%{_docdir}/%{name}/examples
+mkdir -p -m 0755 %{buildroot}/usr/lib/systemd/system
+
+install -m 0444 %{_builddir}/systemd/sybase-dataserver@.service %{buildroot}/usr/lib/systemd/system/
```

Comment:
Why use /usr/lib here and /lib below in restore con?

Reply:
The real dir is `/usr/lib`

The use of `lib` is simpler with the `{}` combination with `/etc`